src/libtriton/bindings/python/namespaces/initRegNamespace.cpp

```diff
@@ -131,6 +131,36 @@ namespace triton {
 
         PyObject* arm32RegistersDictClass = xPyClass_New(nullptr, arm32RegistersDict, xPyString_FromString("ARM32"));
         xPyDict_SetItemString(registersDict, "ARM32", arm32RegistersDictClass);
+
```

Comment:
At top of this file, can you add the doxygen documentation like it is with [others archs](https://github.com/JonathanSalwan/Triton/blob/43d83993fa196b13706f6c2088cce4fdfba4b002/src/libtriton/bindings/python/namespaces/initRegNamespace.cpp#L66-L68)?

To generate the `rv64_reg` and `rv32_reg` files, you have to edit [this one](https://github.com/JonathanSalwan/Triton/blob/43d83993fa196b13706f6c2088cce4fdfba4b002/doc/extract_doc.py). You can simply copy/past the aarch64 part. This python script is used to extract register names from the spec. 

Then, you have to edit this [file](https://github.com/JonathanSalwan/Triton/blob/43d83993fa196b13706f6c2088cce4fdfba4b002/doc/CMakeLists.txt). You can simply copy/past like it is with others archs.